api.go

```diff
@@ -537,6 +538,23 @@ func (s *Stream) Nearest(ctx context.Context, time int64, version uint64, backwa
 	return
 }
 
+// Earliest returns the point nearest to the specified start time searching forward such that the
+// returned point will be >= after. This function is therefore essentially synonymous with the
+// Nearest function.
+func (s *Stream) Earliest(ctx context.Context, after int64, version uint64) (rv RawPoint, ver uint64, err error) {
+	return s.Nearest(ctx, after, version, false)
+}
+
+// Latest returns the point nearest to the specified end time, searching backward such that the
+// returned point will be <= before. If before is not specified, then the current timestamp is used
+// as the starting point to search from.
+func (s *Stream) Latest(ctx context.Context, before int64, version uint64) (rv RawPoint, ver uint64, err error) {
+	if before == 0 {
+		before = time.Now().UnixNano()
```

Comment:
Should we use `1<<61`?

Reply:
I really don't like this. A time series database should not have a notion of "now" and it should not transparently change the result of Latest to being now just because I've loaded data in the future.

Reply:
These are probably the constants you are looking for :
```
const MinimumTime = -(16 << 56)
const MaximumTime = (48 << 56)
```

Reply:
Yeh, agreed, "now" is not useful, I'll update with those constants.